data/tx/legislature/Tan-Parker-fa4817bf-6dc6-4deb-9965-cf3f208ec976.yml

```diff
@@ -1,30 +0,0 @@
-id: ocd-person/fa4817bf-6dc6-4deb-9965-cf3f208ec976
-name: Tan Parker
-given_name: Tan
-family_name: Parker
-gender: Male
-image: https://house.texas.gov/photos/members/4090.jpg
-party:
-- name: Republican
-roles:
-- type: lower
-  jurisdiction: ocd-jurisdiction/country:us/state:tx/government
-  district: '63'
-offices:
-- classification: capitol
-  address: Room 1W.11 P.O. Box 2910 Austin, TX 78768
-  voice: 512-463-0688
-- classification: district
-  address: 800 Parker Square, Suite 245 Flower Mound, TX 75028
```

Comment:
@sroomf this file should be undeleted and their roles should look like this:

- end_date: '2023-01-10'
  type: lower
  jurisdiction: ocd-jurisdiction/country:us/state:tx/government
  district: '63'
- start_date: '2023-01-10'
  type: upper
  jurisdiction: ocd-jurisdiction/country:us/state:tx/government
  district: '12'